packages/components/src/styles/components/layout/flex.scss

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// LAYOUT > FLEX
+//
+
+.hds-layout-flex {
+  display: flex;
+
+  // Note: The gap style is defined here to avoid setting it repeatedly for the gap size variants defined below.
+  // For the gap size variants, we override the value of the gap custom properties to set the desired sizes.
+  // These variables can be used by the consumers as escape hatch if they need to set non-standard gap values (but adds a bit of friction to it)
+  gap:
+    var(--hds-layout-flex-row-gap, 0)
+    var(--hds-layout-flex-column-gap, 0);
+}
+
+// inline
+
+.hds-layout-flex--is-inline {
+  display: inline-flex;
+}
+
+// direction
+
+.hds-layout-flex--direction-row {
+  flex-direction: row;
+}
+
+.hds-layout-flex--direction-column {
+  flex-direction: column;
+}
+
+// wrap
+
+.hds-layout-flex--has-wrapping {
+  flex-wrap: wrap;
+}
+
+// justify
+
+.hds-layout-flex--justify-content-start {
+  justify-content: start;
+}
+
+.hds-layout-flex--justify-content-center {
+  justify-content: center;
+}
+
+.hds-layout-flex--justify-content-end {
+  justify-content: end;
+}
+
+.hds-layout-flex--justify-content-space-between {
+  justify-content: space-between;
+}
+
+.hds-layout-flex--justify-content-space-around {
+  justify-content: space-around;
+}
+
+.hds-layout-flex--justify-content-space-evenly {
+  justify-content: space-evenly;
+}
+
+.hds-layout-flex--justify-content-stretch {
+  justify-content: stretch;
+}
+
+// align
+
+.hds-layout-flex--align-items-start {
+  align-items: start;
+}
+
+.hds-layout-flex--align-items-center {
+  align-items: center;
+}
+
+.hds-layout-flex--align-items-end {
+  align-items: end;
+}
+
+.hds-layout-flex--align-items-stretch {
+  align-items: stretch;
+}
+
+// gap
+
+// set just row gap size
```

Comment:
```suggestion
// set row gap size
```

Reply:
I am going to remove the comment at all, it's redundant at this point